src/Lykke.Service.OAuth/Controllers/AuthenticationController.cs

```diff
@@ -288,6 +298,40 @@ public async Task<VerificationCodeResult> VerifyEmail([FromBody] VerificationCod
             return result;
         }
 
+        /// <summary>
+        /// Check if email is already registered
+        /// </summary>
+        /// <param name="model"></param>
+        /// <response code="200">Validation result</response>
+        /// <response code="400">Email hash is invalid, BCrypt work factor is invalid</response>
+        [HttpPost]
+        [Route("~/registration/email")]
+        [SwaggerOperation("ValidateEmail")]
+        [ProducesResponseType(typeof(ErrorResponse), (int) HttpStatusCode.BadRequest)]
+        [ProducesResponseType(typeof(EmailValidationResult), (int) HttpStatusCode.OK)]
```

Comment:
Better to move successful response Attribute before unsuccessful ones.
```suggestion
        [ProducesResponseType(typeof(EmailValidationResult), (int) HttpStatusCode.OK)]
        [ProducesResponseType(typeof(ErrorResponse), (int) HttpStatusCode.BadRequest)]
```